config.sample.yaml

```diff
@@ -206,6 +206,9 @@ ircService:
         # not apply an idle timeout. This value is ignored if this IRC server is
         # mirroring matrix membership lists to IRC. Default: 172800 (48 hours)
         idleTimeout: 10800
+        # The number of millseconds to wait between consecutive reconnections if a
+        # client gets disconnected. Default: 5000 (5 seconds)
```

Comment:
You don't mention how to disable this (setting 0? -1?).

I also thought we decided to have a "base" reconnect time to which this value will be added to it, yet this description makes no mention of this.